docs/source/supergraphs.md

```diff
@@ -76,11 +97,21 @@ rover supergraph compose --config ./supergraph.yaml > prod-schema.graphql
 
 #### Gateway compatibility
 
-The `rover supergraph compose` command produces a supergraph schema by using composition functions from the [`@apollo/subgraph`](/federation/api/apollo-subgraph/) package. Because that library is still in pre-1.0 releases (as are Rover and Apollo Gateway), some updates to Rover might result in a supergraph schema with new functionality. In turn, this might require corresponding updates to your gateway.
+**Apollo Gateway fails to start up if it's provided with a supergraph schema that it doesn't support.** To ensure compatibility, we recommend that you test launching your gateway in a CI pipeline with the supergraph schema it will ultimately use in production.
+
+The `rover supergraph compose` command generates a supergraph schema by dynamically selecting the proper composition function either from the [`@apollo/federation`](https://www.npmjs.com/package/@apollo/federation) package for Federation 1, or from the [`@apollo/composition`](https://www.npmjs.com/package/@apollo/composition) library for Federation 2. Rover works with _multiple_ versions of composition without needing to build and release a separate version of Rover.
```

Comment:
sure. it is _kinda_ dynamic, since if you set it to `2` it will grab the latest in the `2` channel and if you set it to `1` it will grab the latest in the `1` channel. maybe not necessary to include the dynamic word though

Reply:
There's probably an awesome buzzword for something that's quasi-dynamic

Reply:
completely unrelated to this, but related to awesome buzzwords: i learned that [epigraph](https://en.wikipedia.org/wiki/Epigraph_(mathematics)) is a synonym for supergraph in mathematics

Reply:
🧠